Projects/hGraphQL/GraphQL/Contracts.graphql

```diff
@@ -0,0 +1,119 @@
+query Contracts($locale: Locale!) {
+  contracts {
+    id
+    displayName
+    switchedFromInsuranceProvider
+    gradientOption
+    upcomingRenewal {
+      renewalDate
+      draftCertificateUrl
+    }
+    perils(locale: $locale) {
+      ...PerilFragment
+    }
+    statusPills(locale: $locale)
+    detailPills(locale: $locale)
+    insurableLimits(locale: $locale) {
+      ...InsurableLimitFragment
+    }
+    currentAgreementDetailsTable(locale: $locale) {
+      ...DetailsTableFragment
+    }
+    upcomingAgreementDetailsTable(locale: $locale) {
+      ...DetailsTableFragment
+    }
+    termsAndConditions(locale: $locale) {
+      displayName
+      url
+    }
+    ...UpcomingAgreementFragment
+    currentAgreement {
+      ... on AgreementCore {
+        certificateUrl
+        activeFrom
+        activeTo
+        status
+        premium {
+          ...MonetaryAmountFragment
+        }
+      }
+      ... on SwedishApartmentAgreement {
```

Comment:
We shouldn't use these, only fetch on `AgreementCore` to avoid tight coupling with specific contract types